lib/ig.js

```diff
@@ -774,7 +774,10 @@ ${match[1]}
         pushXmlResource(`${exResource.resourceType}/${exResource.id}`, name, 'example');
 
         if (exResource.resourceType === 'SearchParameter') {
-          const baseTypes = exResource.base.join(", ");
+          let baseTypes = exResource.base;
+          if (exResource.base.constructor === Array) {
```

Comment:
In our case, this is probably fine, but in general `Array.isArray(foo)` is preferred over `foo.constructor === Array` -- as it works in all circumstances.  Plus I think it's easier to read.

See: https://stackoverflow.com/questions/28467266/can-i-use-obj-constructor-array-to-test-if-object-is-array

Also, I might have considered writing this whole thing as:
```js
const baseTypes = Array.isArray(exResource.base) ? exResource.base.join(',') : exResource.base;
```
But that is truly a matter of preference.  Some people hate the ternary operator.  I don't know if you are one of those people. ;-)

Reply:
I am not a hater, so I fixed it to use `Array.isArray()` and the ternary operator.